modyn/config/schema/pipeline.py

```diff
@@ -312,7 +318,6 @@ class TrainingConfig(BaseModel):
         description="The number of data loader workers on the trainer node that fetch data from storage.", ge=1
     )
     batch_size: float = Field(description="The batch size to be used during training.", ge=1)
-    learning_rate: float = Field(0.1, description="The learning rate to be used during training.")
```

Comment:
`learning_rate` is not controlled by this parameter. They are specified per para groups in `param_groups`.